config/crd/bases/bootstrap.cluster.x-k8s.io_kzerosworkerconfigs.yaml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
We need to regenerate install.yaml to include the new CRDs and the changes in `config/rbac/role.yaml`.

I believe `make release` will take care of everything without further changes to the Makefile or `config/default/kustomization.yaml`

Reply:
In #117 we removed it from the repo to avoid confusion. It is still generated as part of the release and the docs refer to it as:
```
kubectl apply -f https://docs.k0smotron.io/stable/install.yaml
```

Reply:
Ah that's smart.